src/.env.sample

```diff
@@ -147,5 +147,14 @@ RESOURCE_TYPES="projects,observations,observation_with_rubrics,surveys,programs"
 #maximum number of tasks allowed in a project
 MAX_PROJECT_TASK_COUNT= 10
 
+#consumption portal name ex: self/diksha/elevate-project
+CONSUMPTION_SERVICE=self
+
+#Publish method to consumption side kafka/api
+PUBLISH_METHOD=kafka
+
+#kafka topic for publish
+PROJECT_PUBLISH_KAFKA_TOPIC=publishtopic
```

Comment:
@priyanka-TL Let's introduce Kafka message on/off like we have here and based on that topic is optional.https://github.com/project-sunbird/ml-core-service/blob/release-8.0.0/.env.sample